midp/gfx.js

```diff
@@ -441,28 +443,18 @@ var currentlyFocusedTextEditor;
         return [x, y];
     }
 
-    function measureWidth(c, str) {
-        return c.measureText(str).width | 0;
-    }
-
-    function withFont(font, c) {
-        if (c.font != font.css) {
-          c.font = font.css;
-        }
-    }
-
     // withTextAnchor() can be hot. This variable lets it return two values
     // without creating a new array every time.
     var withTextAnchorRet = [0, 0];
 
     function withTextAnchor(g, c, anchor, x, y, str) {
-        withFont(g.currentFont, c);
+        c.font = g.currentFont.css;
 
         c.textAlign = "left";
         c.textBaseline = "top";
 
         if (anchor & RIGHT || anchor & HCENTER) {
-            var w = measureWidth(c, str);
+            var w = c.measureText(str).width;
```

Comment:
The width used to be truncated with "| 0". Integer coordinates used to be faster than floating point coordinates on mobile, I'm not sure if this is still true.